templates/frontend/pages/catalogCategory.tpl

```diff
@@ -39,7 +39,7 @@
 	<div class="about_section{if $image} has_image{/if}{if $description} has_description{/if}">
 		{if $image}
 			<div class="cover" href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="fullSize" type="category" id=$category->getId()}">
-				<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="category" id=$category->getId()}" alt="{$category->getLocalizedTitle()|escape}" />
+				<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="category" id=$category->getId()}" alt="{$category->getLocalizedTitle()|default:'null'}" />
```

Comment:
We still need to escape the title to prevent a XSS attack. In this case (and below), you can "chain" the filters together like this:

```
{$category->getLocalizedTitle()|escape|default:'null'}
```